static/scss/_button.scss

```diff
@@ -263,6 +263,7 @@
   font-weight: 400;
   line-height: 36px;
   border-radius: 2px;
+  height: fit-content;
```

Comment:
@annagav would you like to add a screen with more list items along the button ?

Reply:
Ok, I looked around the `fit-content` property and seems good to me in this scenario but would be nice if you add a screen with more list items along the button